modules/openapi-generator/src/main/resources/cpp-pistache-server/main-api-server.mustache

```diff
@@ -53,7 +53,7 @@ int main() {
     std::vector<int> sigs{SIGQUIT, SIGINT, SIGTERM, SIGHUP};
     setUpUnixSignals(sigs);
 #endif
-    Pistache::Address addr(Pistache::Ipv4::any(), Pistache::Port({{serverPort}}));
+    Pistache::Address addr(Pistache::Ipv4::any(), Pistache::Port({{#serverPort}}{{serverPort}}{{/serverPort}}{{^serverPort}}8080{{/serverPort}}));
```

Comment:
if you change stuff for pistache as well here, then you have to regenerate the samples for pistache as well

Reply:
I did, there is no change. 
The method produces default 8080 `serverPort` value, now it does not so that we don't need to append port for clients.
Check the removal of code from the pistache java file.

Reply:
I also regenerated Qt5 server, which was not updated before.